cogs/misc.py

```diff
@@ -134,8 +136,77 @@ async def calc(
         if chart_constant is None and rating > 0:
             sign = "+"
 
+        res = f"Rating: {sign}{floor_to_ndp(rating, 2)}"
+        if chart_constant is not None:
+            overpower_max = calculate_overpower_max(chart_constant)
+            if score == 1010000:
+                res += f"\nOVER POWER: {floor_to_ndp(overpower_max, 2)} / {floor_to_ndp(overpower_max, 2)} (100.00%)"
+            elif score < 500000:
+                res += f"\nOVER POWER: 0.00 / {floor_to_ndp(overpower_max, 2)} (0.00%)"
+            else:
+                overpower_base = calculate_overpower_base(score, chart_constant)
+                res += f"\nOVER POWER:"
+                if score >= 1000000:
+                    overpower = overpower_base + Decimal(1)
+                    res += f"\n• AJ: {floor_to_ndp(overpower, 2)} / {floor_to_ndp(overpower_max, 2)} ({floor_to_ndp(overpower / overpower_max * 100, 2)}%)"
+                overpower = overpower_base + Decimal(0.5)
+                res += f"\n• FC: {floor_to_ndp(overpower, 2)} / {floor_to_ndp(overpower_max, 2)} ({floor_to_ndp(overpower / overpower_max * 100, 2)}%)"
+                res += f"\n• Non-FC: {floor_to_ndp(overpower_base, 2)} / {floor_to_ndp(overpower_max, 2)} ({floor_to_ndp(overpower_base / overpower_max * 100, 2)}%)"
+
         await ctx.reply(
-            f"Calculation result: {sign}{floor_to_ndp(rating, 2)}", mention_author=False
+            res, mention_author=False
+        )
+
+    @commands.hybrid_command("const", aliases=["constant"])
+    async def const(
+        self, ctx: Context, chart_constant: float
+    ):
+        """Calculate rating and over power achieved with various scores based on chart constant.
+
+        Parameters
+        ----------
+        chart_constant: float
+            Chart constant of the chart. Use the `info` command to find this.
+        """
+
+        if chart_constant < 1 or chart_constant > 16:
+            raise commands.BadArgument("Chart constant must be between 1.0 and 16.0")
+
+        scores = [1009900, 1009500, 1009000, 1008500, 1008000, 1007500, 1007000, 1006500, 1006000, 1005500, 1005000, 1004000, 1003000, 1002000, 1001000, 1000000, 997500, 995000, 992500, 990000, 987500, 985000, 982500, 980000, 977500, 975000, 970000, 960000, 950000, 925000, 900000]
+        res = "```  Score |  Rate |      OP | OP (FC) | OP (AJ)\n----------------------------------------------"
+        rating = calculate_rating(1010000, chart_constant)
+        if rating < 10:
+            rating_pad = " "
+        else:
+            rating_pad = ""
+        res += f"\n1010000 | {rating_pad}{floor_to_ndp(rating, 2)} |       - |       - | 100.00%"
```

Comment:
Python f-strings support padding:
```suggestion
        res += f"\n1010000 | {floor_to_ndp(rating, 2):>4} |       - |       - | 100.00%"
```